.mvn/develocity.xml

```diff
@@ -19,25 +19,22 @@
     under the License.
 
 -->
-<gradleEnterprise
-        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
-  <!-- Enable Gradle Enterprise extension when GRADLE_ENTERPRISE_ACCESS_KEY is set and the build isn't
+<develocity xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
+<!-- Enable Gradle Enterprise extension when GRADLE_ENTERPRISE_ACCESS_KEY is set and the build isn't
        a pull request from a branch or forked repository with a name that indicates it's a work in progress. -->
   <enabled>#{env['GRADLE_ENTERPRISE_ACCESS_KEY']?.trim() > '' and !(env['GITHUB_HEAD_REF']?.matches('(?i).*(experiment|wip|private).*') or env['GITHUB_REPOSITORY']?.matches('(?i).*(experiment|wip|private).*'))}</enabled>
   <server>
     <url>https://ge.apache.org</url>
     <allowUntrusted>false</allowUntrusted>
+    <accessKey>#{env['GRADLE_ENTERPRISE_ACCESS_KEY'].substring(env['GRADLE_ENTERPRISE_ACCESS_KEY'].indexOf('=') + 1)}</accessKey>
   </server>
   <buildScan>
     <capture>
-      <goalInputFiles>true</goalInputFiles>
       <buildLogging>true</buildLogging>
       <testLogging>true</testLogging>
     </capture>
     <backgroundBuildScanUpload>#{isFalse(env['GITHUB_ACTIONS'])}</backgroundBuildScanUpload>
-    <publish>ALWAYS</publish>
-    <publishIfAuthenticated>true</publishIfAuthenticated>
```

Comment:
nit: I am wondering why we need to update these configs? Could you explain a little here?

Reply:
[followed the migration guide](https://docs.gradle.com/develocity/maven-extension/legacy/#develocity_migration) and checked in IntelliJ that the new XML was following the new XML schema. These elements have been removed / replaced and that's why I removed them.